yocto-dev/build_conf/local.conf

```diff
@@ -0,0 +1,41 @@
+MACHINE ??= 'ucm-imx93'
+DISTRO ?= 'fsl-imx-xwayland'
+PACKAGE_CLASSES ?= 'package_rpm'
+EXTRA_IMAGE_FEATURES ?= "debug-tweaks"
+USER_CLASSES ?= "buildstats"
+PATCHRESOLVE = "noop"
+BB_DISKMON_DIRS ??= "\
+    STOPTASKS,${TMPDIR},1G,100K \
+    STOPTASKS,${DL_DIR},1G,100K \
+    STOPTASKS,${SSTATE_DIR},1G,100K \
+    STOPTASKS,/tmp,100M,100K \
+    HALT,${TMPDIR},100M,1K \
+    HALT,${DL_DIR},100M,1K \
+    HALT,${SSTATE_DIR},100M,1K \
+    HALT,/tmp,10M,1K"
+PACKAGECONFIG:append:pn-qemu-system-native = " sdl"
+CONF_VERSION = "2"
+
+DL_DIR ?= "${BSPDIR}/downloads/"
+ACCEPT_FSL_EULA = "1"
+
+# Switch to Debian packaging and include package-management in the image
+PACKAGE_CLASSES = "package_deb "
+EXTRA_IMAGE_FEATURES += "package-management"
+LICENSE_FLAGS_ACCEPTED = "commercial"
+
+CORE_IMAGE_EXTRA_INSTALL += " cl-uboot cl-deploy cl-growfs-rootfs "
+CORE_IMAGE_EXTRA_INSTALL += " libubootenv-bin u-boot-compulab-env "
+CORE_IMAGE_EXTRA_INSTALL += " memtester htop iotop tmux iperf3 "
+CORE_IMAGE_EXTRA_INSTALL += " libgpiod-tools bluez5 can-utils minicom eeprom-util usbutils i2c-tools "
+CORE_IMAGE_EXTRA_INSTALL += " wireless-tools dhcpcd ifupdown wpa-supplicant ntp ntpdate "
+CORE_IMAGE_EXTRA_INSTALL += " networkmanager modemmanager "
+
+# additional recipes for Ice Cream
```

Comment:
Same comment about code name as before. Also be more specific about how this is being used. Example: Recipes for Kimera-VIO and its dependencies.